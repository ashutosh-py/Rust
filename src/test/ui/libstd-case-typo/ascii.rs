// checks case typos with libstd::ascii structs
fn main(){}

fn test_escdef(_x: Escapedefault){}
//~^ ERROR: cannot find type `Escapedefault` in this scope
