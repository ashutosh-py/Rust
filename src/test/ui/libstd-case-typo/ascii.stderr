error[E0412]: cannot find type `Escapedefault` in this scope
  --> $DIR/ascii.rs:4:20
   |
LL | fn test_escdef(_x: Escapedefault){}
   |                    ^^^^^^^^^^^^^ not found in this scope
   |
help: did you mean one of these?:
   |
LL | fn test_escdef(_x: std::ascii::EscapeDefault){}
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^
LL | fn test_escdef(_x: std::char::EscapeDefault){}
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^
LL | fn test_escdef(_x: std::str::EscapeDefault){}
   |                    ^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0412`.
